mirage/ramp_generator/obs_generator.py

```diff
@@ -2829,7 +2829,8 @@ def save_DMS(self, ramp, zeroframe, filename, mod='1b', err_ext=None,
         with warnings.catch_warnings():
             warnings.simplefilter("ignore")
             current_time = datetime.datetime.utcnow()
-            start_time_string = self.params['Output']['date_obs'] + 'T' + self.params['Output']['time_obs']
+            # start_time_string = str(self.params['Output']['date_obs']) + 'T' + str(self.params['Output']['time_obs'])
+            start_time_string = '2022-08-27T12:49:00.000'
```

Comment:
Did you mean to leave this in?

Reply:
No! This needs correcting. `self.params['Output']['date_obs']` is a `astropy.time.Time` object and I was having trouble making it into an appropriately formatted string. I'll fix it!

Reply:
Ok I figured out this was because I needed to put the date and time strings in quotes in the yaml file. It's fixed now.